src/query/query_hash_scan.h

```diff
@@ -41,6 +41,16 @@
     } \
   while (0)
 
+/* kind of hash list scan method */
+enum hash_method
+{
+  NOT_USE = 0,
+  IN_MEMORY = 1,
+  HYBRID_IN_MEMORY = 2,
+  HASH_TEMP_FILE = 3 /* not used */
```

Comment:
To recognize the enum value is related to the hash method, how about adding a prefix for example HASH_METH_IN_MEM?

Reply:
okay.